src/common/SPELLS/classic/warrior.ts

```diff
@@ -1,7 +1,262 @@
 import { spellIndexableList } from '../Spell';
 
 const spells = spellIndexableList({
-  VIGILANCE: {
+  // --------
+  // SHARED
+  // --------
+ 
+Battle_Shout: {
+    id: 47436,
+    name: 'Battle Shout (Rank 9)',
+    icon: 'ability_warrior_battleshout', 
+   },
```

Comment:
Each spell with multiple ranks needs a `lowRanks` attribute with the lower rank ids in an array. For example:
```suggestion
    lowRanks: [2048, 25289, 11551, 11550, 11549, 6192, 5242, 6673],
   },
```